control/server.py

```diff
@@ -514,6 +501,12 @@ def _create_transport(self, trtype):
 
     def keep_alive(self):
         """Continuously confirms communication with SPDK process."""
+        self.allowed_consecutive_spdk_ping_failures = self.config.getint_with_default("gateway",
+                                                                                      "allowed_consecutive_spdk_ping_failures", 1)
+        self.spdk_ping_interval_in_seconds = self.config.getfloat_with_default("gateway", "spdk_ping_interval_in_seconds", 2.0)
+        if self.spdk_ping_interval_in_seconds < 0.0:
+            self.logger.warning(f"Invalid SPDK ping interval {self.spdk_ping_interval_in_seconds}, will reset to 0")
+            self.spdk_ping_interval_in_seconds = 0.0
```

Comment:
No need to use "self." for these fields as they are only used within this function

Reply:
See [diff](https://github.com/ceph/ceph-nvmeof/compare/4650717dcc6c12c2e409610bc955854419b85640..8228fef6a48994808779b19af96a06a765970f42)